com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/json/RestUtils.java

```diff
@@ -136,7 +136,8 @@ private static MTable getTable(String tableName) {
 		if (table != null && table.isView() && tableName.toLowerCase().endsWith("_v")) {
 			boolean hasVT = DB.isTableOrViewExists(tableName+"t");
 			if (hasVT) {
-				table = MTable.get(Env.getCtx(), tableName+ "t");
+				MTable t_table = MTable.get(Env.getCtx(), tableName+ "t");
```

Comment:
what's the goal of this change?
the variable hasVT already confirmed that the table _vt exists, so it's not necessary to check again for its existence